### PR TITLE
changed home page name

### PR DESCRIPTION
**Pull request by ophy90**
J'ai corrigé le nom de la page home. Merci de valider ma modification et merger cette branche :-)

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -16,7 +16,5 @@
     <%= render 'shared/navbar' %>
     <%= render 'shared/flashes' %>
     <%= yield %>
-    <p class="notice"><%= notice %></p>
```
Comment (Altrngo) [inline app/views/layouts/application.html.erb:19]: Notice and alert class have been removed, could you give some details ?

**Review by Altrngo — COMMENTED** [review]
Notice and alert class have been removed, could you give some details ?

File: app/views/layouts/application.html.erb (hunk 1 above)
Comment (ophy90) [reply app/views/layouts/application.html.erb:19]: The notice and alert are included in the shared / flashes.

Comment (Altrngo) [reply app/views/layouts/application.html.erb:19]: Indeed ! very nice job! 👯

**Pull request merged**